src/components/StateManagement.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {stateLoad} from './../actions';
+import {URL} from './../services';
+
+const encoding = 'UTF-8';
+
+class StateManagement extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: props.state
+    };
+    this.save = this.save.bind(this);
+    this.load = this.load.bind(this);
+    this.fileInput = {click: () => null};
+  }
+
+  render() {
+    return <div>
+      <input ref={input => this.fileInput = input} type="file" onChange={this.load} hidden="hidden"/>
+      <button onClick={this.save}>Save</button>
+      <button onClick={() => this.fileInput.click()}>Load</button>
+    </div>
+  }
+
+  save() {
+    const json = JSON.stringify(this.state.data);
+    const blob = new Blob([json], {encoding, type: `text/json;charset=${encoding}`});
+    const url = URL.createObjectURL(blob);
```

Comment:
[Experimental tech](https://developer.mozilla.org/en-US/docs/Web/API/Window/URL), but available in all modern browsers.